theano/gpuarray/tests/test_pickle.py

```diff
@@ -29,7 +29,7 @@
         cuda_ndarray.use('gpu', default_to_move_computation_to_gpu=False,
                          move_shared_float32_to_gpu=False,
                          enable_cuda=False)
-        theano.sandbox.gpuarray.init_dev('cuda')
```

Comment:
This is seem a good code cleanup for another PR.

On Fri, May 6, 2016 at 3:31 PM, abergeron <notifications@github.com> wrote:

> In theano/gpuarray/tests/test_pickle.py
> <https://github.com/Theano/Theano/pull/4454#discussion_r62379432>:
>
> > @@ -29,7 +29,7 @@
> >          cuda_ndarray.use('gpu', default_to_move_computation_to_gpu=False,
> >                           move_shared_float32_to_gpu=False,
> >                           enable_cuda=False)
> > -        theano.sandbox.gpuarray.init_dev('cuda')
>
> I mean that this should use either the normal code to get the test context
> or just initiailize straight away and not go through cuda_ndarray (which
> doesn't work anymore).
>
> I remember that this was done this way because if you import config it
> raises SkipTest when pygpu is not available. I think that catching SkipTest
> and noting that is a better way than what is here.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/4454/files/a5b12a87e8389f7d78a7b51d541ae840854b3fc1#r62379432>
>